### PR TITLE
DOCS-6043-HTTP-responseTimeout-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Per feedback received updating default Mule configuration response timeout

### CODE REVIEW

**Review by emariotti3 — APPROVED** [review]

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 HTTP Connector v1.5
 
-Anypoint Connector for HTTP (HTTP Connector) is used to handle and perform HTTP requests. This class 
+Anypoint Connector for HTTP (HTTP Connector) is used to handle and perform HTTP requests. This class
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
Use Anypoint Connector for HTTP (HTTP Connector) to handle and perform HTTP requests. This class
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 2]
```diff
@@ -99,7 +99,7 @@ Configuration element for a HTTP requests.
 ** ALWAYS
 ** NEVER |  Whether to specify a correlationId when publishing messages. This applies both for custom correlation ids specifies at the operation level and for default correlation Ids taken from the current event |  AUTO |
 | Preserve Headers Case a| Boolean |  By default, header keys are stored internally in lower-case. This is to improve performance of headers handling and is functionally correct as specified in the RFC. <p> In the case a server expects headers in a specific case, this flag may be set to true so the case of the header keys are preserved. |  false |
-| Response Timeout a| Number |  Maximum time in milliseconds that the request element will block the execution of the flow waiting for the HTTP response. If this value is not present, the default response timeout from the Mule configuration will be used. | 10000 |
+| Response Timeout a| Number |  Maximum time in milliseconds that the request element will block the execution of the flow waiting for the HTTP response. If this value is not present, the default response timeout from the Mule configuration will be used, which is also 10000 | 10000 |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Response Timeout a| Number |  Maximum time in milliseconds that the request element blocks the execution of the flow waiting for the HTTP response. If this value is not present, the connector uses the default response timeout from the Mule configuration, which is also 10000 | 10000 |
```
Please check if this is accurate.

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 3]
```diff
@@ -358,11 +358,11 @@ Represents a listener for HTTP requests.
...
-| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  | 
-| Password a| String | The password used to protect the trust store. |  | 
-| Type a| String | The type of store used. |  | 
-| Algorithm a| String | The algorithm used by the trust store. |  | 
-| Insecure a| Boolean | If true, no certificate validations will be performed, rendering connections vulnerable to attacks. Use at your own risk. |  | 
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
+| Password a| String | The password used to protect the trust store. |  |
+| Type a| String | The type of store used. |  |
+| Algorithm a| String | The algorithm used by the trust store. |  |
+| Insecure a| Boolean | If true, no certificate validations will be performed, rendering connections vulnerable to attacks. Use at your own risk. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Insecure a| Boolean | If true, no certificate validations are performed, rendering connections vulnerable to attacks. Use at your own risk. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 4]
```diff
@@ -371,12 +371,12 @@ Represents a listener for HTTP requests.
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the key store. |  | 
-| Type a| String | The type of store used. |  | 
-| Alias a| String | When the key store contains many private keys, this attribute indicates the alias of the key that should be used. If not defined, the first key in the file will be used by default. |  | 
-| Key Password a| String | The password used to protect the private key. |  | 
-| Password a| String | The password used to protect the key store. |  | 
-| Algorithm a| String | The algorithm used by the key store. |  | 
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the key store. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Path a| String | The location (which resolves relative to the current classpath and file system, if possible) of the key store. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 5]
```diff
@@ -371,12 +371,12 @@ Represents a listener for HTTP requests.
...
+| Type a| String | The type of store used. |  |
+| Alias a| String | When the key store contains many private keys, this attribute indicates the alias of the key that should be used. If not defined, the first key in the file will be used by default. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Alias a| String | When the key store contains many private keys, this attribute indicates the alias of the key that should be used. If not defined, the first key in the file is used by default. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 6]
```diff
@@ -416,9 +416,9 @@ Represents a listener for HTTP requests.
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 7]
```diff
@@ -495,15 +495,15 @@ Represents a listener for HTTP requests.
...
-| Initial Buffer Size a| Number | This is the amount of memory that will be allocated in order to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then it expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
-| Buffer Size Increment a| Number | How much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower will mean that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised when the buffer gets full. |  | 
-| Max Buffer Size a| Number | This is the maximum amount of memory that will be used. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised. A value lower or equal to zero means no limit. |  | 
+| Initial Buffer Size a| Number | This is the amount of memory that will be allocated in order to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then it expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Initial Buffer Size a| Number | This is the amount of memory allocated in order to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 8]
```diff
@@ -495,15 +495,15 @@ Represents a listener for HTTP requests.
...
+| Buffer Size Increment a| Number | How much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower will mean that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised when the buffer gets full. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc:499]:
```suggestion
| Buffer Size Increment a| Number | Specifies how much the buffer size expands if the buffer exceeds its initial size. Setting a value of zero or lower prevents the buffer from expanding its size. A STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 9]
```diff
@@ -495,15 +495,15 @@ Represents a listener for HTTP requests.
...
+| Max Buffer Size a| Number | This is the maximum amount of memory that will be used. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error will be raised. A value lower or equal to zero means no limit. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Max Buffer Size a| Number | This is the maximum amount of memory that the buffer can use. If the buffer uses more than that, a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower or equal to zero means there's no limit. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 10]
```diff
@@ -512,13 +512,13 @@ Represents a listener for HTTP requests.
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Max In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed then it will start to buffer the content on disk. |  | 
+| Max In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed then it will start to buffer the content on disk. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Max In Memory Size a| Number | Defines the maximum memory that the stream can use to keep data in memory. If more than that is consumed, the content is buffered on disk. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 11]
```diff
@@ -527,11 +527,11 @@ Represents a listener for HTTP requests.
...
-| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  | 
-| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  | 
-| Object Store a| ObjectStore | The object store where the redelivery counter for each message is going to be stored. |  | 
+| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering process-failed-message |  |
+| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message |  |
+| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  |
+| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  |
+| Object Store a| ObjectStore | The object store where the redelivery counter for each message is going to be stored. |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Object Store a| ObjectStore | The object store where the redelivery counter for each message is stored. |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 12]
```diff
@@ -646,9 +646,9 @@ Represents a listener for HTTP requests.
 | Field | Type | Description | Default Value | Required
 | Host a| String | Host where the proxy requests will be sent. |  | x
 | Port a| Number | Port where the proxy requests will be sent. |  | x
-| Username a| String | The username to authenticate against the proxy. |  | 
-| Password a| String | The password to authenticate against the proxy. |  | 
-| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy should not be used |  | 
+| Username a| String | The username to authenticate against the proxy. |  |
+| Password a| String | The password to authenticate against the proxy. |  |
+| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy should not be used |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy is not used |  |
```

File: modules/ROOT/pages/http/http-documentation.adoc [hunk 13]
```diff
@@ -660,9 +660,9 @@ Represents a listener for HTTP requests.
 | Ntlm Domain a| String | The domain to authenticate against the proxy. |  | x
 | Host a| String | Host where the proxy requests will be sent. |  | x
 | Port a| Number | Port where the proxy requests will be sent. |  | x
-| Username a| String | The username to authenticate against the proxy. |  | 
-| Password a| String | The password to authenticate against the proxy. |  | 
-| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy should not be used |  | 
+| Username a| String | The username to authenticate against the proxy. |  |
+| Password a| String | The password to authenticate against the proxy. |  |
+| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy should not be used |  |
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/http/http-documentation.adoc]:
```suggestion
| Non Proxy Hosts a| String | A list of comma separated hosts against which the proxy is not used |  |
```

**Review by Cristian-Venticinque — APPROVED** [review]
Please check suggestions before merging

**Pull request merged**